src/fileformats/ocd_file_import.cpp

```diff
@@ -1076,6 +1085,59 @@ void OcdFileImport::setupBaseSymbol(Symbol* symbol, const OcdBaseSymbol& ocd_bas
 	symbol->setIsHelperSymbol(false);
 	symbol->setProtected(ocd_base_symbol.status & Ocd::SymbolProtected);
 	symbol->setHidden(ocd_base_symbol.status & Ocd::SymbolHidden);
+	
+	if (ocd_version == 8 && ocd_base_symbol.flags & 0x02)
+	{
+		addWarning(tr("In symbol symbol %1 '%2': %3").
+		           arg(symbol->getNumberAsString(), symbol->getName(),
+		               tr("Unsupported compressed icon")));
+#ifdef MAPPER_DEVELOPMENT_BUILD
+		qDebug("Unsupported compressed symbol icon: %s", qPrintable(symbol->getName()));
+		/// \todo Factor out generic hex dump function
+		constexpr auto extent = std::extent<typename std::remove_pointer<decltype(ocd_base_symbol.icon.bits)>::type>::value;
+		auto icon_bits = ocd_base_symbol.icon.bits;
+		for (int i = 0; i < 16; ++i)
+		{
+			QByteArray data;
+			data.reserve(80);
+			data.append(QByteArray("000")
+			            .append(QByteArray::number(16*i,16))
+			            .right(4)
+			            .append("   "));
+			for (int j = 0; j < 17; ++j)
+			{
+				if (icon_bits == ocd_base_symbol.icon.bits + extent)
+					break;
+				if (*icon_bits < 0x10)
+					data += '0';
+				data += QByteArray::number(*(icon_bits++), 16) + ' ';
+			}
+			qDebug("%s", data.constData());
+		}
+#endif
+	}
+	else
+	{
+		symbol->setCustomIcon(importSymbolIcon(ocd_base_symbol.icon));
+	}
+}
+
+
+template<class OcdBaseSymbol>
+void OcdFileImport::checkSymbolIcon(Symbol* symbol, const OcdBaseSymbol& ocd_base_symbol)
```

Comment:
`checkSymbolIcon()` does not express what the function is doing. `tryIconReplace()` might be a better fit.

Reply:
How about `dropRedundantIcon()`?

Reply:
`dropRedundantIcon()` sounds good to me.